build.gradle

```diff
@@ -18,8 +18,13 @@ plugins {
 	id 'com.github.johnrengelman.shadow' version '8.1.1'
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
I believe vscode/eclipse still needs these lines to work?

Reply:
Does someone have eclipse to test this? Toolchains are at this point not a new gradle feature; I'd be rather concerned if eclipse didn't support them.

Reply:
This was addressed on discord: it works fine, luckily.